clio-client/src/main/scala/org/broadinstitute/clio/client/parser/BaseArgs.scala

```diff
@@ -5,7 +5,9 @@ import java.time.OffsetDateTime
 import org.broadinstitute.clio.client.ClioClientConfig
 import org.broadinstitute.clio.client.commands.Commands
 
-case class BaseArgs(command: Option[String] = None,
+import scala.concurrent.ExecutionContext
+
+case class BaseArgs(command: Option[Commands.Command] = None,
                     bearerToken: String = "",
                     metadataLocation: String = "",
```

Comment:
Should `metadataLocation` be an `Option` too, since this class is serving as the bucket-of-all-args that might or might not be needed?